src/app/charts/core/templates/_helpers-sso.tpl

```diff
@@ -267,8 +267,10 @@ idp.xml
         {{- $protocol = ($saml | ternary "saml" "oidc") -}}
       {{- end -}}
 
+      {{- $baseUrl := (include "arkcase.tools.conf" (dict "ctx" $ctx "value" "baseUrl")) -}}
+
```

Comment:
Remember to remove the trailing slashes:

`$baseUrl = (regexReplaceAll "/*$" $baseUrl "")`